lib/firebase_options.dart

```diff
@@ -0,0 +1,76 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      return web;
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for macos - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.linux:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for linux - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions web = FirebaseOptions(
+    apiKey: 'AIzaSyC0EyoArHl-pu2FOLLuMiomkWyGxFx_KWk',
+    appId: '1:346027065713:web:5b385a5c51417f54d636d0',
+    messagingSenderId: '346027065713',
+    projectId: 'stargreen-starfield',
+    authDomain: 'stargreen-starfield.firebaseapp.com',
+    storageBucket: 'stargreen-starfield.appspot.com',
+    measurementId: 'G-56B87Y9ZNX',
+  );
+
+  static const FirebaseOptions android = FirebaseOptions(
+    apiKey: 'AIzaSyDp6BXjZ_-mR-0UBZPYO9trYQImGI-qqEA',
+    appId: '1:346027065713:android:76c0d89dc1445ef7d636d0',
+    messagingSenderId: '346027065713',
+    projectId: 'stargreen-starfield',
+    storageBucket: 'stargreen-starfield.appspot.com',
+  );
+
```

Comment:
we should hide this with env variables 👀

Reply:
Yeah, i will create some flavors for avoid use it directly. I'm thinking to migrate all auth with Laravel for our own database and api with GraphQL.